api-documentation.js

```diff
@@ -452,6 +446,21 @@ class ApiDocumentation extends EventsTargetMixin(AmfHelperMixin(LitElement)) {
     this.requestUpdate('serversCount', old);
   }
 
+  get shouldRenderSelector() {
+    const { noServerSelector, narrow } = this;
+
+    return !noServerSelector && narrow;
```

Comment:
This can return `undefined` but it should return `boolean`. I understand that this is the same from the program PoV but make it explicit by adding `!!` to `narrow`:
```
return !noServerSelector && !!narrow;
```

Reply:
got it